db/migrate/20180614094306_add_user_id_for_activities.rb

```diff
@@ -0,0 +1,23 @@
+class AddUserIdForActivities < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :activities, :user, index: true
+
+    reversible do |dir|
+      dir.up do
+        Activity.find_each do |activity|
+          id = User.find_by_email(activity.read_attribute(:user)).id
```

Comment:
what if the User doesn't exist in the DB anymore? This would cause an `undefined method on Nil` error

Reply:
Come to think of it, what if they still exist but they've changed their email address?